tests/unit/service_test.py

```diff
@@ -786,11 +786,11 @@ def setUp(self):
         self.mock_client = mock.create_autospec(docker.Client)
 
     def test_build_volume_binding(self):
-        binding = build_volume_binding(VolumeSpec.parse('/outside:/inside'))
+        binding = build_volume_binding(VolumeSpec.parse('/outside:/inside', True))
```

Comment:
Does the `normalize` argument need to be included here (and in the other calls in the file)? It seems irrelevant.